### ISSUE TITLE
vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php on line 93

### ISSUE THREAD

**Issue by connectjames** [magento/magento2#7072]
### Preconditions
1. 2.0.9 -> 2.1.2 (upgrade)
2. mysql 5.6, php 7.0.6, cPanel
### Steps to reproduce
1. Went through the wizard
2. All good and green to go
3. Finished the upgrade
### Expected result
1. Go to the website page and see the website working
### Actual result

Error: 
Parse error: syntax error, unexpected '.' in /home/longstone/public_html/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php on line 93

Everything in SSH works correctly, upgrade, compile, clear cache, reindex...

**Comment by connectjames** [comment]
If somebody come across this error, you must have forgot to add the add to you htaccess file.

AddType application/x-httpd-php76 .php

Thanks

**Comment by suntrop** [comment]
I ~~am~~ was using PHP 5.5.33 and ~~get~~ got the same error. Now with 5.6.16 it works.
Perhaps this helps people coming from google :-)

**Comment by suntrop** [comment]
Ok, 2.1 doesn't work on PHP 5.5. 2.0 was the last release working with 5.5

**Comment by ut4utc** [comment]
doesn't work on PHP 7.0.8

**Comment by manthanthakker** [comment]
Whats the solution then?

**Comment by coatesjonathan** [comment]
whats the solution?

**Comment by snez** [comment]
+1

**Comment by snez** [comment]
Solved: The command "php bin/magento deploy:mode:show" was failing on PHP 5.5.9-1ubuntu4.20 (cli) but is succeeding with PHP 7.0.12 (cli)

**Comment by max133** [comment]
Got this error in my running Magento 2.1.4 one time in the middle of the night 0:00 as a cronjob error from:
php7.0 /var/www/html/magento-ds/bin/magento cron:run | grep -v "Ran jobs by schedule" >> /var/www/html/magento-ds/var/log/magento.cron.log


> PHP Fatal error:  Uncaught Error: Class 'Codazon\AjaxLayerNav\Model\Cron' not found in /var/www/html/magento-ds/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php:93
> Stack trace:
> #0 /var/www/html/magento-ds/vendor/magento/framework/ObjectManager/Factory/Compiled.php(88): Magento\Framework\ObjectManager\Factory\AbstractFactory->createObject('Codazon\\AjaxLay...', Array)
> #1 /var/www/html/magento-ds/vendor/magento/framework/ObjectManager/ObjectManager.php(57): Magento\Framework\ObjectManager\Factory\Compiled->create('Codazon\\AjaxLay...', Array)
> #2 /var/www/html/magento-ds/vendor/magento/module-cron/Observer/ProcessCronQueueObserver.php(235): Magento\Framework\ObjectManager\ObjectManager->create('Codazon\\AjaxLay...')
> #3 /var/www/html/magento-ds/vendor/magento/module-cron/Observer/ProcessCronQueueObserver.php(195): Magento\Cron\Observer\ProcessCronQueueObserver->_runJob(1491955200, 1491955201, Array, Object(Magento\Cron\Model\Schedule), 'default')
> #4 /var/www/html/magento-ds/vendor/magento/framework/Event/Invoker/ in /var/www/html/magento-ds/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php on line 93

PHP 7.0.15-0ubuntu0.16.04.4 (cli) ( NTS )
Mysql 5.7.17

**Comment by Gujarats** [comment]
I'm having the same problem like this too.I'm using this Ubuntu 16.04 php 7.0xx.
I solve the problem by the following command : 

```shell
$ bin/magento setup:upgrade
$ bin/magento setup:di:compile
```
Now the error is gone. But not sure why this is happen though.

**Comment by kristinabuntmetall** [comment]
I had the same Problem, a new Magento 2.1.7 installation, php 7.0.2, mysql 5.6. in developer mode and the cronjob always send me an email with: 
_Class 'IntlDateFormatter' not found in /vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php on line 93_

But I had to get the shop online and changed to production mode, ignoring the error, because I found no solution (but updatin the system) into the www. But I had no more time for that.
I switched both entries for the base-urls (secure und unsecure) under "stores > configuration > general > web" to my secure https://domain.com, because I did that with older Magento-installations also.

Now everything works fine.
